### PR TITLE
Update issue templates #2

### PR DESCRIPTION
**Pull request by Boccca2014**

### CODE REVIEW

**Review by JJamesWWang — COMMENTED** [review]
How come you want to remove a lot of the helper text?

**Comment by Boccca2014** [comment]
> How come you want to remove a lot of the helper text?

I feel like it's too many words on the page, but I'll add it back if you think it will be more helpful.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,30 @@
...
-labels: ''
+labels: 'bug'
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Describe the Bug**
+
```
Comment (JJamesWWang) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: Might want to keep this for consistency sake if every other section has helper text, but I think this one's up to you

**Review by JJamesWWang — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Boccca2014) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]: Alright added it all back but added an extra newline to give them some extra room to view each section separately.

**Review by JJamesWWang — APPROVED** [review]

**Pull request merged**